src/app/components/Pagination/Pagination.jsx

```diff
@@ -27,10 +27,11 @@ class Pagination extends React.Component {
     const intPage = parseInt(page, 10);
     const pageNum = type === 'Next' ? intPage + 1 : intPage - 1;
     const svg = type === 'Next' ? <RightArrowIcon /> : <LeftArrowIcon />;
+    const url = this.props.createAPIQuery({ page: pageNum });
```

Comment:
I think `url` will at least be `"q="` based on the function `basicQuery` in `utils`.  So `<Link>`'s `to` will either have a valid path or `/search?$q=`. It probably won't cause issues but maybe not the best practice?